src/app/templates/templates-routes.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { TemplatesModule } from './templates.module';
+import { TemplatesListPage } from './list-page/list-page.component';
+
+const routes: Routes = [
+  { path: '', component: TemplatesListPage, pathMatch: 'full' },
+];
+
+@NgModule({
+  imports: [
+    RouterModule.forChild(routes),
+    TemplatesModule,
+  ],
+})
+export class TemplateRoutesModule { }
```

Comment:
I'm not a fan of the filename but I'm struggling to come up with an alternative :-)

It feels redundant, and it almost feels like it should be `template.routes.ts` but then I get that since it contains an angular module it'd be good to reflect that in the name.